### PR TITLE
PR to merge new routes into server

### PR DESCRIPTION
**Pull request by tjspitz**
**This PR contains routes authored by Kev:**
- /reviews
- /reviews/:review_id/helpful
- /reviews/:review_id/report
- /reviews?product_id=40344&page=1&count=5&sort=helpful

**Merged into this branch ( `rel_items` ), which contains routes authored by Tom:**
- /products/:product_id/related
- /interactions

**Approved PR will result in:**
- branch `main` now contains all previously-authored routes
- other authors can now pull `main` to merge, push, and PR to add their routes

**Other notes:**
- all routes above were successfully tested via Postman
- local `.eslintrc.js` may need reinstalling after pulling & merging on your local

### CODE REVIEW

File: server/controllers/interactions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        Authorization: process.env.API_KEY,
+      },
+    };
+    axios(options)
+      .then((result) => {
+        res.status(201).json(result.data);
+      })
+      .catch((err) => {
+        console.log(err);
+        res.sendStatus(422);
```
Comment (quanjingchen) [inline server/controllers/interactions.js:29]: i'm wondering why the status code for failed post request is 422

**Review by quanjingchen — COMMENTED** [review]
nice job!!! i just have a minor question about the status code.

File: server/controllers/interactions.js (hunk 1 above)
Comment (tjspitz) [reply server/controllers/interactions.js:29]: If I am understanding the Learn page, that error code is given as the appropriate response:

image

Comment (quanjingchen) [reply server/controllers/interactions.js:29]: i see. thanks!

**Pull request merged**